outputs.tf

```diff
@@ -24,8 +29,8 @@ output "cluster_version" {
 }
 
 output "cluster_security_group_id" {
-  description = "Security group ID attached to the EKS cluster."
-  value       = local.cluster_security_group_id
+  description = "Cluster security group ID attached to the EKS cluster."
+  value       = aws_eks_cluster.this[0].vpc_config[0].cluster_security_group_id
```

Comment:
Can you test this output with a 1.13 cluster? The EKS controlled cluster security group is only added for kubenetes 1.14 and eks.3 platform. I'm wondering what Terraform sets the value to or whether it's an error.

Also, this pattern won't work with `create_eks = false` or possibly with a half destroyed state. See the `concat(aws[*].output, [""])[0]` patterned used in other outputs.

Reply:
The first issue exists today.    Renaming the variables here didn't change anything besides the output.   

I fixed the concat but haven't pushed it up yet because I am trying to work a solution for the 1.13 issue but now that I have tested,  the issue exists in the current release as well as this change.

Reply:
in a 1.13 cluster, the security_group_id is the primary field but still exists (either created or passed in), in a 1.14+ cluster, the security_group_id still either exists or is created but is just used as the additional field.  I added a check to only use attempt to create node groups with 1.14 or greater.    The eks.3 version doesn't matter because if you try to create a node group going forward, it will use the latest version (eks.8) and if you have an existing and try to modify it, it will force a recreate and pick up the latest.